frontend/src/libs/components/profile-settings-sidebar/profile-settings-sidebar.tsx

```diff
@@ -0,0 +1,59 @@
+import { useCallback, useState } from 'react';
+
+import { type IconNames } from '#libs/enums/enums.js';
+
+import { Icon } from '../components.js';
+import { ProfileSettingsButton } from './profile-settings-button/profile-settings-button.js';
+import styles from './styles.module.scss';
+
+const ProfileSettingsSidebar: React.FC = () => {
+  const [activeButton, setActiveButton] = useState<string>('');
+
+  const handleOnClick = useCallback((name: IconNames) => {
+    setActiveButton(name);
+  }, []);
+
+  return (
+    <div className={styles['container']}>
+      <div className={styles['header']}>
+        <div className="visually-hidden">Profile settings</div>
```

Comment:
@fshabanov Do we need here import scaffolding.scss and use <styles file>['<className>'] notation?

Reply:
Well, I just followed the general approach for other components that were merged before

Reply:
we don't need to import `scaffolding.scss` as it is available globally